api/URL.json

```diff
@@ -970,10 +970,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "11"
```

Comment:
https://mdn-bcd-collector.appspot.com/tests/api/URL/toJSON tests the prototype only. I think this is probably right still, but I'd be more comfortable with testing an instance, which is easy in this case.

Reply:
Although the test is only testing the prototype, this is correct; `toJSON()` isn't defined on the instance until Safari 11.

Reply:
Thanks for confirming!